.azure-pipelines/ci.yml

```diff
@@ -17,7 +17,20 @@ jobs:
       displayName: 'Post Analysis'
       inputs:
         CredScan: true
-
+  - job: CodeQL
+    displayName: Code QL
+    pool:
+      vmImage: 'windows-latest'
+    steps:
+      - task: Semmle@1
+        inputs:
+          sourceCodeDirectory: '$(Build.SourcesDirectory)'
+          language: 'tsandjs'
```

Comment:
Will you enable it on Java?

Reply:
good question, let me try if I can enable java together.

Reply:
seems we cannot set multiple languages for the same task, so I added another task for java. hope it works.